stdlib/3/urllib/response.pyi

```diff
@@ -31,12 +33,19 @@ class addbase(BinaryIO):
     def write(self, s: bytes) -> int: ...
     def writelines(self, lines: Iterable[bytes]) -> None: ...
 
+class addclosehook(addbase):
+    closehook: Callable
+    hookargs: Tuple[Any]
+    def __init__(self, fp: BinaryIO, closehook: Callable, *hookargs: Any) -> None: ...
+
 class addinfo(addbase):
     headers: Message
+    def __init__(self, fp: BinaryIO, headers: Message) -> None: ...
     def info(self) -> Message: ...
 
 class addinfourl(addinfo):
     url: str
     code: int
+    def __init__(self, fp: BinaryIO, headers: Message, url: str, code: Optional[int] = None) -> None: ...
```

Comment:
```suggestion
    def __init__(self, fp: BinaryIO, headers: Message, url: str, code: Optional[int] = ...) -> None: ...
```

Reply:
Stubs generally use `...` as the form of default arguments, no matter their runtime value

Reply:
Not sure how I forgot that. Thanks!